WrightTools/data/_data.py

```diff
@@ -215,14 +212,16 @@ def min(self):
 class Channel:
     """Channel."""
 
-    def __init__(self, values, units=None, file_idx=None, null=None, signed=None, name='channel',
-                 label=None, label_seed=None):
+    def __init__(self, values, name, units=None, file_idx=None, null=None, signed=None,
```

Comment:
file_idx is intended to be removed in a future commit, correct?

Reply:
thanks